### PR TITLE
add default storage config

### PR DESCRIPTION
**Pull request by cleverhu**

### CODE REVIEW

File: pkg/storage/internalstorage/register.go [hunk 1]
```diff
@@ -66,6 +68,17 @@ func NewStorageFactory(configPath string) (storage.StorageFactory, error) {
...
 	}
+	sqlDB, err := db.DB()
+	if err != nil {
+		return nil, err
+	}
+	connPool, err := cfg.getConnPoolConfig()
+	if err != nil {
+		return nil, err
+	}
+	sqlDB.SetMaxIdleConns(connPool.MaxIdleCount)
```
Comment (Iceber) [inline pkg/storage/internalstorage/register.go:79]: typo: MaxIdleConns

File: pkg/storage/internalstorage/register.go [hunk 2]
```diff
@@ -66,6 +68,17 @@ func NewStorageFactory(configPath string) (storage.StorageFactory, error) {
...
+	sqlDB.SetMaxOpenConns(connPool.MaxOpenConns)
+	sqlDB.SetConnMaxLifetime(connPool.MaxLifetime)
```
Comment (Iceber) [inline pkg/storage/internalstorage/register.go:81]: Is consistent with the `SetConnMaxLifetime` function, named ConnMaxLifetime?

Comment (cleverhu) [reply pkg/storage/internalstorage/register.go]: OK, I also think it's easier to understand by this.

**Review by Iceber — APPROVED** [review]
LGTM

**Comment by Iceber** [comment]
We need to support the configuration of conn poll in helm charts

**Pull request merged**